runtime/compiler/x/codegen/J9CodeGenerator.cpp

```diff
@@ -126,6 +126,12 @@ J9::X86::CodeGenerator::initialize()
       cg->setSupportsStackAllocationOfArraylets();
       }
 
+   static bool disableInlineVectorHashCode = feGetEnv("TR_disableInlineVectorHashCode") != NULL;
+   if (comp->target().cpu.supportsFeature(OMR_FEATURE_X86_SSE4_1) && !disableInlineVectorHashCode)
+      {
+      cg->setSupportsInlineVectorizedHashCode();
```

Comment:
Can you confirm you've tested this at least internally on 32-bit?  I don't feel particularly strongly about optimizing on the 32-bit path, so if you are unable to test and wanted to limit to 64-bit then I'm fine with that.

Reply:
The `vectorizedHashcode(...) ` API doesn't exist on older JDK levels for which we have 32-bit builds. Do you want me to exclude 32-bit anyway?

Reply:
Ah...  No, that's OK.  It will become a testing exercise for someone wanting to bring 32-bit support to one of these JDK levels later.

Reply:
Shouldn't this be disabled if arraylets are enabled?

Reply:
Yes, since it can't handle discontiguous array shapes.